test/src/UpdateIdAndMyselfTest.cc

```diff
@@ -86,7 +86,7 @@ namespace {
 
             char* start_username = NULL;
             char* myself_name = NULL;
-            const char* own_user_id = "Own_Beavis";
```

Comment:
> What?

What is the TODO here?

Reply:
I am mostly confused as to why exactly was the own_user_id set ever and at all here.
I assume this must've an old code left over or similar since you've removed it anyways.

Reply:
CORE-217 disallows own user ids like this, that's why I removed this. Before, the core didn't care much, but there could be confusion with other users and also with the signing identity, which could prevent you from being able to use your own user id.